src/main/resources/config.properties

```diff
@@ -0,0 +1,2 @@
+source.path=src/data/fruit_shop_transactions
+target.path=src/reports/report
```

Comment:
move the files to the resources directory. I don't recommend complicating your solution using a file with variables.